src/main/kotlin/com/pubnub/api/PubNub.kt

```diff
@@ -2283,15 +2313,32 @@ class PubNub internal constructor(
         return UuidMetadataImpl(this, ChannelName(id))
     }
 
-    fun channelGroup(name: String): ChannelGroup {
-        return ChannelGroupImpl(this, ChannelGroupName(name))
-    }
-
+    /**
+     * Create a [SubscriptionSet] from the given [subscriptions].
+     *
+     * @param subscriptions the subscriptions that will be added to the returned [SubscriptionSet]
+     * @return a [SubscriptionSet] containing all [subscriptions]
+     */
     fun subscriptionSetOf(subscriptions: Set<Subscription> = emptySet()): SubscriptionSet {
         @Suppress("UNCHECKED_CAST")
         return SubscriptionSetImpl(this, subscriptions as Set<SubscriptionImpl>)
     }
 
+    /**
+     * Create a [SubscriptionSet] containing [Subscription] objects for the given sets of [channels] and
+     * [channelGroups].
+     *
+     * Please note that the subscriptions are not active until you call [SubscriptionSet.subscribe].
+     *
+     * This is the same as calling [PubNub.channel] followed by [Channel.subscription] for each channel,
```

Comment:
This sentence is not clear for me. Maybe because I am new to new version of Event Listeners.

Reply:
modified it a bit, PTAL

Reply:
changing it a bit to make it clearer